package.json

```diff
@@ -19,7 +19,8 @@
         "test:ci": "npm run test:unit && npm run test:e2e:ci",
         "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
 	"format": "prettier --write src",
-	"format-lint": "npm run format && npm run lint",
+	"format-test": "prettier --write tests",
+	"format-lint": "npm run format && npm run format-test && npm run lint",
```

Comment:
I think you can add the `tests` folder directly into `format` rule, no need for a specific `format-test` rule

Reply:
and you should maybe run prettier once on the package.json file directly, you are using tabs instead of spaces 😉